File: dayfour/get_perfect_square.py
def get_square():
	test_list = [4,6,49,25]
	
	array = [0,0,0,0]
	for counting in range(0,4):
		count = 0
		number = test_list[counting]
		
		counter = 0

		while number > 0:
			
			if counter % 2 > 0:
				number-=counter
				count+=1
				
			counter+=1
		print(count)
		if count*count == test_list[counting]: 
			array[counting]=True
				
		else:
			array[counting]=False
		if test_list[counting]==0:
			array[counting]=False
				

	return array

print(get_square())
		
		
	

File: dayfour/get_perfect_square.js
function get_perfect_square(test_list){
	let array = []
	for (let count = 0; count < test_list.length; count++) {
		let counting = 0;
		
		let number=test_list[count];
		for(let counter = 0;number > 0; counter++){
			if (counter % 2 > 0){
				number-=counter;
				counting++;
			} 
			}
			
			if(counting*counting == test_list[count]) array[count]=true;	
			else array[count]=false;
			if (test_list[count]==0)array[count]=false;
				
			
			}
return array;
}
console.log(get_perfect_square([1,0,4,100,101]))
